lib/bitcoin/BitcoinClient.ts

```diff
@@ -89,14 +92,44 @@ export default class BitcoinClient {
     const transaction = await this.createTransaction(transactionData, feeInSatoshis);
     const rawTransaction = transaction.serialize();
 
-    const request = {
-      method: 'sendrawtransaction',
-      params: [
-        rawTransaction
-      ]
-    };
+    return this.broadcastTransactionRpc(rawTransaction);
+  }
 
-    return this.rpcCall(request, true);
+  /**
+   * Broadcasts the specified lock transaction.
+   *
+   * @param bitcoinLockTransaction The transaction object.
+   */
+  public async broadcastLockTransaction (bitcoinLockTransaction: BitcoinLockTransactionModel): Promise<string> {
+    if (!(bitcoinLockTransaction.transactionObject instanceof Transaction)) {

```

Comment:
Why not just force it to be a Transaction object instead of Any?

Reply:
Don't want to expose the bitcore-lib object outside of this class really.